SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -244,10 +246,10 @@ SCDocHTMLRenderer {
 		<< "</script>\n"
 		<< "<script src='" << baseDir << "/scdoc.js' type='text/javascript'></script>\n"
 		<< "<script src='" << baseDir << "/docmap.js' type='text/javascript'></script>\n" // FIXME: remove?
-		<< "</head>\n";
-
-		stream
-		<< "<body onload='fixTOC()'>\n";
+		// QWebChannel access
+		<< "<script src='qrc:///qtwebchannel/qwebchannel.js' type='text/javascript'></script>\n"
+		<< "</head>\n"
+		<< "<body onload='didLoadRenderedWindow()'>\n";
```

Comment:
this is more of a problem with the existing code, but `<body onload="...">` is really old hat. it's better to leave `fixTOC` be, and then write this in the JS:

```
$(function () {
    if (typeof QWebChannel !== "undefined") {
        setUpWebChannel();
    }
});
```

Reply:
OK, where should I write that?

Reply:
it should go in `scdoc.js`, at the top level. `<body onload="fixTOC()">`, and the `fixTOC()` function itself, should stay as is.